src/forbes.py

```diff
@@ -0,0 +1,39 @@
+"""Forbes 40 billionaires list."""
+
+from flight_path import get_data_from_json_file
+
+
+def youngest_and_oldest_under_eighty():
+    """Using forbes data return the youngest and oldest under 80.
+
+    Challenge, not using sort function.
+    """
+    data = get_data_from_json_file('rich-ppl.json')
+
+    min_age = {'age': 100}
+    max_age = {'age': 20}
+
+    for idx in data:
+        if 0 < idx['age'] < min_age['age']:
+            min_age = idx
+        if max_age['age'] < idx['age'] < 80:
+            max_age = idx
+
+    min_age.pop('country', None)
+    max_age.pop('country', None)
+    min_age.pop('rank', None)
+    max_age.pop('rank', None)
```

Comment:
Interesting way to handle this, popping out the data you don't need.  This seems completely fine for this assignment, but I think that if you had more 'junk data', it might be easier to just try and extract what you need and package that up to be returned.